ocfweb/account/templates/account/vhost_mail/index.html

```diff
@@ -113,12 +117,61 @@
                         <span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
                         Add address
                     </button>
+                    <h4>CSV import/export</h4>
+                    <form class="form-inline js-import-csv-form" enctype="multipart/form-data" action="{% url 'vhost_mail_csv_import' vhost.domain %}" method="post">
+                        <a class="btn btn-default" href="{% url 'vhost_mail_csv_export' vhost.domain %}">
+                            <span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span>
+                            Download CSV
+                        </a>
+                        <div class="form-group">
+                            <label class="btn btn-default">
+                                <span class="glyphicon glyphicon-arrow-up" aria-hidden="true"></span>
+                                Upload CSV
+                                <input type="file" class="hidden js-import-csv" name="csv_file" accept="text/csv" />
+                            </label>
+                        </div>
+                        <a href="#csv_import_export">What's this?</a>
+                        {% csrf_token %}
+                    </form>
                 {% endwith %}
             {% endfor %}
             <hr />
             <small>
                 * Passwords can't be retrieved after being set, but you can always change them again.
             </small>
+            <div class="row">
+                <div class="col-md-8">
+                    <h4 id="csv_import_export">Importing and exporting CSVs</h4>
+                    <p>
+                        You can use CSV spreadsheets to manage your forwarding
+                        addresses. Clicking "Download CSV" will allow you to
+                        download all your forwarding addresses for a particular
+                        domain and the emails they forward to. Clicking "Upload
+                        CSV" will let you pick a CSV file to add new forwarding
+                        rules or update existing ones; no addresses will be
+                        deleted.
+                    </p>
+                    <p>
+                        The CSV format for downloading and uploading has two
+                        columns. The first contains the name part of the
+                        forwarding address (i.e. the part before
+                        <code>@mygroup.berkeley.edu</code>). The second
+                        contains all the receiving emails in a list. When
+                        uploading, the list may be separated by any combination
+                        of commas or whitespace, as long as the column is
+                        quoted when line breaks or commas are used.
+                    </p>
```

Comment:
I'm concerned that this paragraph might fly over the heads of a large chunk of our target audience.
Perhaps explain what the CSV should look like in a spreadsheet program? (Two columns, left column is ..., right column is ..., addresses in right column can be separated by comma or whitespace, yadda yadda)

Reply:
That's a good suggestion. I think this image will do the trick:

![example_csv](https://user-images.githubusercontent.com/6722367/31048703-b370c3e8-a5d7-11e7-8378-43993427fa3b.png)

Reply:
I feel like we can entirely omit the technical specification of the CSV format and simply show the image of what the CSV file looks like in a spreadsheet program.

Reply:
I can make it somewhat shorter, but I don't think the text description should be entirely omitted. What if someone using a screen reader visits the page? Plus, it's important for people uploading CSVs, so the information bears repeating.